utils/mantra_post_test_results

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+
+set -e
+set -x
```

Comment:
:warning: Will this leak the `MANTRA_PASSWORD`?

Reply:
This is pre-existing, and I don't see it in the CI output, so it should be all right.